mk/config.mk

```diff
@@ -661,3 +661,17 @@ CFG_COMPAT_GP10_DES ?= y
 
 # Defines a limit for many levels TAs may call each others.
 CFG_CORE_MAX_SYSCALL_RECURSION ?= 4
+
+# Pseudo-TA to export hardware RNG output to Normal World
+# RNG characteristics are platform specific
+ifeq ($(CFG_HWRNG_PTA),y)
```

Comment:
add a default value for CFG_HWRNG_PTA before this test:
```
CFG_HWRNG_PTA ?= n
```

Reply:
Maintainer question: should we adopt this as the preferred way? Undefined also means disabled. But `n` has the advantage of being explicitly saved in the generated build configuration file (`$(out-dir)/conf.mk`) which means that if the default ever changes to `y`, the old configuration could still be used with `make CFG_OPTEE_CONFIG=path/to/saved_conf.mk` and effectively disable the feature.

Reply:
I'm not sure how useful it is to be able to supply an old `conf.mk`, but I like the idea of assigning default disabled config variables. It makes the description of the variable more natural.

Reply:
OK let's do that.

Reply:
done